packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx

```diff
@@ -0,0 +1,40 @@
+# Punctuation
+
+## Use periods consistently
+
+Keep UX copy scannable by eliminating unnecessary periods.
+Do not use periods on:
+
+- buttons
```

Comment:
I feel like this would read better if it wasn't a list.

> **Do not** use periods on buttons, labels, tooltips, or bulleted lists. **Do** use periods when using multiple sentences, any sentence followed by a link, or modal body text.